packages/client/src/contexts/Lang.re

```diff
@@ -15,4 +15,5 @@ let measurableIsSaved = "Question is saved.";
 let copiedMessage = "Copied";
 let copiedDescription = "The question ID was copied.";
 let backButtonTip = "Back";
-let pageIsNotFound = "Sorry, but the page is not found.";
\ No newline at end of file
+let pageIsNotFound = "Sorry, but the page is not found.";
+let notebookRemove = "Are you sure delete this notebook?";
```

Comment:
Change to "Are you sure you want to delete this notebook?"

Reply:
Thanks.